boilerplates/be-docker-plain/files/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# add /overwrite this with your dockerfile ..
+# FROM nginx:1.15.0-alpine
```

Comment:
Should we leave the default `FROM` instructions so that the first deployment "works"?

Reply:
yes, I agree on leaving the *FROM* and keeping a note

Reply:
Both agreed- Ill add it.

Reply:
ok you convinced me .. FROM registry.access.redhat.com/rhel7 :)